releases/note-source/v15_0.rst

```diff
@@ -0,0 +1,239 @@
+-----------------------------------------
+- :ref:`release-v15-0-diffim`
+- :ref:`release-v15-0-mem`
+- :ref:`release-v15-0-artifacts`
+- :ref:`release-v15-0-skysub`
+- :ref:`release-v15-0-wcs`
+- :ref:`release-v15-0-firefly`
+- :ref:`release-v15-0-compression`
+- :ref:`release-v15-0-cmdline`
+- :ref:`release-v15-0-prereqs`
+
+
+
+.. _release-v15-0-diffim:
+
+Image differencing algorithm improvements
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Algorithms appropriate for differencing in a variety of contexts have been implemented as tasks in the LSST stack. These various algorithms and configurations have been compared.  The results are written up in `DMTN-061 <https://dmtn-061.lsst.io>`_.
+
+.. _release-v15-0-mem:
+
+Performance improvements in coaddition
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Now all coaddition algorithms have significantly reduced memory footprints
+
+.. _release-v15-0-artifacts:
+
+Significantly improved artifact rejection in coaddition
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Coaddition algorithms that do artifact clipping can now handle artifacts that overlap from epoch to epoch.
+
+.. _release-v15-0-skysub:
+
+Full focal plane sky subtraction
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+There are now tasks to create and apply models of the sky that extend over the entire field of view. View notes on `CLO <https://community.lsst.org/t/sky-subtraction/2415>`_.
+
+.. _release-v15-0-wcs:
+
+Replace all ``Wcs`` classes with the `AST <https://arxiv.org/abs/1602.06681>`_ backed ``SkyWcs``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The last release introduced a transform system backed by the AST package. Since that release the stack has been converted to using that system in all contexts where a world coordinate system is required.
+
+.. _release-v15-0-firefly:
+
+Plotting frontend for Firefly
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This release includes, for the first time, the package that allows the LSST plotting abstraction layer to plot directly in the Science Portal plotting tool, Firefly. View on `GitHub <https://github.com/lsst/display_firefly>`_.
+
+.. _release-v15-0-compression:
+
+Lossless compression on by default
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Lossless compression is turned on by default when persisting any image-like data product. Read the `decision <https://jira.lsstcorp.org/browse/RFC-378>`_ and view the notes on `CLO <https://community.lsst.org/t/lossless-fits-compression-enabled/2410>`_ and links therein.
+
+.. _release-v15-0-cmdline:
+
+Changes to command line tasks
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Command line tasks now handle clobbering of versions, data, and configs in a more intuitive way. E.g. output repositories are now expected to differ from the input repository.  This eliminates the need to explicitly turn on clobbering when making multiple runs to different outputs (reruns) when using the same inputs.
```

Comment:
Can we link to docs on this?

Reply:
I meant to.  I'll figure out where the clo post is.